### DISCUSSION TITLE
Ready for the Party :-)

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#307] [Show and tell]
Here it is the Party ready Photobooth.

BOM:
Raspberry pi 4 4GB
Nikon D3100 ( no liveview )
Web Cam
HDMI Switch ( as the autofocus of the Nikon took too long I connect it also via HDMI and do manual focus)
Old Notbook monitor with an lcd controller card
Canon Selphy CP1500 
Travel Access point to save the performance of the PI
Box : V-Solt 2020 profile and plastic sheets
LED Display : ESP32 2 x MAX7219 8x8
Switch : ESP32 , 3 butons, case self designed and printed 
Deco : 3d printed

Next is test test test ;-) 

I stared in March with the first tests and prototype.

Thanks to Mike for the great work and support.
![IMG_7663](https://github.com/photobooth-app/photobooth-app/assets/37927465/4b1a8f7b-999e-473f-90b9-b00a06dbd7f7)

**Comment by mgineer85** [comment]
Awesome!
